lib/ansible/cli/arguments/option_helpers.py

```diff
@@ -210,7 +210,7 @@ def create_base_parser(prog, usage="", desc=None, epilog=None):
 def add_verbosity_options(parser):
     """Add options for verbosity"""
     parser.add_argument('-v', '--verbose', dest='verbosity', default=C.DEFAULT_VERBOSITY, action="count",
-                        help="verbose mode (-vvv for more, -vvvv to enable connection debugging)")
+                        help="verbose mode (set from -v to -vvvvvv, -vvvv to enable connection debugging)")
```

Comment:
actually, that is not really the limit (just the limit of internally created aliases), verbosity does not have a hardcoded limit at this point (indirectly python int max value)

Reply:
@bcoca thanks for pointing that out!

Can we assume that Ansible  is not drastically increasing output when adding more than 6 `v` regarding the following line?

```
elif level in ('debug', 'v', 'vv', 'vvv', 'vvvv', 'vvvvv', 'vvvvvv'):
```

I also searched through the code and found 6 as the max used as well, might be better hidden.

Even if 6 is not the absolute limit, it might still be good to hint that at least level 6 has en effect, what do you think?

Reply:
afaik, winrm is only one using v X 6 and no one is doing the `display.verbose( ..., caplevel=23)`, I was pointing it out JIC you wanted a more precise wording.

Reply:
@bcoca the style of the different help messages is not uniform, but I try something, please feel free to come up with a more precise wording.

```
Causes Ansible to print more debug messages. Adding multiple -v will increase the verbosity, the currently used maximum for build in plugins is -vvvvvv. Connection debugging requires at least -vvvv.)
```

Reply:
I really hate that is the case, we never got a real good standard around 'number of v' to use for each case, most connections print all output at 3, winrm is one of the few that goes higher

Reply:
@bcoca a clear guideline what every level unveils would be awesome, but that is not trivial.

```
plugins/lookup/__init__.py
123:                self._display.warning("Unable to find '%s' in expected paths (use -vvvvv to see paths)" % needle)
```

From the help I had the impression -vvvv would be the maximum used and only learned otherwise after getting a friendly hint to use -vvvvv.
That is the sub problem I want to fix here. 😄 
What do you think about the wording, I added your last input?

```txt
Causes Ansible to print more debug messages. Adding multiple -v will increase the verbosity, the build in plugins currently evaluate up to -vvvvvv. A reasonable level to start is -vvv, connection debugging might require -vvvv.)
```